docfx_project/api/index.md

```diff
@@ -0,0 +1,108 @@
+# Puppeteer Sharp

```

Comment:
isn't the content of this file already in `docfx_project/index.md`?

Reply:
docfx_project/index.md => Site home => https://www.puppeteersharp.com/
docfx_project/api/index.md => It's the API home  => https://www.puppeteersharp.com/api/

Reply:
can we embed it into the other page?